### ISSUE TITLE
Treeview Item height

### ISSUE THREAD

**Issue by pierresa** [Microsoft/Windows-universal-samples#538]
I am trying to have a rather dense tree without too much space between line items. I have suppressed any margin definitions, added padding="0 everywhere, added a container style without any margin/padding,  built my own IntegerToIndentationConverter to make sure the Thickness top/bottom were set to 0 ... anyway it does not change the layout; each line item is over 20 height while my Items are only 12.

is there a solution to fix that?

Thank you
Pierre

**Comment by juanpaexpedite** [comment]
What part are you changing the ItemTemplate o the ItemContainerStyle?

**Comment by pierresa** [comment]
Thank you for your reply.

I changed the TreeViewItemDataTemplate (removing any margin command) , I added a container style:

<Style TargetType="tvs:TreeViewItem" x:Key="ContainerStyle">
            <Setter Property="Margin" Value="0" />
            <Setter Property="Padding" Value="0"/>
            <Setter Property="Height" Value="12"/>
        </Style>

And nothing changes in the line spacing for the tree.

Thank you
Pierre
De : juanpaexpedite [mailto:notifications@github.com]
Envoyé : 26 November 2016 18:25
À : Microsoft/Windows-universal-samples <Windows-universal-samples@noreply.github.com>
Cc : pierresa <pierresa@outlook.com>; Author <author@noreply.github.com>
Objet : Re: [Microsoft/Windows-universal-samples] Treeview Item height (#538)


What part are you changing the ItemTemplate o the ItemContainerStyle?
